pkg/controller/remotemachineset/remotemachineset_controller.go

```diff
@@ -128,6 +129,10 @@ func (r *ReconcileRemoteMachineSet) Reconcile(request reconcile.Request) (reconc
 		log.WithError(err).Error("error looking up cluster deployment")
 		return reconcile.Result{}, err
 	}
+	if cd.Annotations != nil && cd.Annotations[constants.AnnotationSyncsetPause] == "true" {
```

Comment:
You probably want this to be the following for more flexibility.
```
if p, err := strconv.ParseBool(cd.Annotations; p && err != nil {
``

Reply:
@wanghaoran1988 This comment was probably missed, the above format would be a little more flexible and let you be less strict on the format of "true" in the annotation.